drivers/mpu9250/MPU9250.hpp

```diff
@@ -178,8 +178,13 @@ namespace DriverFramework
 #define M_PI_F 3.14159265358979323846f
 #endif
 
+#if defined(__EDISON)
+// update frequency 100 Hz
+#define MPU9250_MEASURE_INTERVAL_US 10000
```

Comment:
Also, note that you have to adapt the code here depending on how fast the _measure() loop here runs:
https://github.com/PX4/Firmware/blob/master/src/platforms/posix/drivers/df_mpu9250_wrapper/df_mpu9250_wrapper.cpp#L609